draft-ietf-acme-acme.md

```diff
@@ -2583,26 +2597,34 @@ Required parameters: None
 
 Optional parameters: None
 
-Encoding considerations: None
+Encoding considerations: 7bit
 
-Security considerations: Carries a cryptographic certificate and its associated certificate chain
+Security considerations: Carries a cryptographic certificate and its associated certificate chain.  This media type carries no active content.
 
 Interoperability considerations: None
 
 Published specification: draft-ietf-acme-acme
 \[\[ RFC EDITOR: Please replace draft-ietf-acme-acme above with the RFC number assigned to this ]]
 
-Applications which use this media type: Any MIME-compliant transport
+Applications which use this media type: ACME clients and servers, HTTP servers, other applications that need to be configured with a certificate chain
 
 Additional information:
 
-File contains one or more certificates encoded with the PEM textual encoding, according to
-RFC 7468 {{!RFC7468}}.  In order to provide easy interoperation with TLS, the first
-certificate MUST be an end-entity certificate. Each following certificate
-SHOULD directly certify the one preceding it. Because certificate validation
-requires that trust anchors be distributed independently, a certificate
-that specifies a trust anchor MAY be omitted from the chain, provided
-that supported peers are known to possess any omitted certificates.
+  Deprecated alias names for this type: n/a
+  Magic number(s): n/a
+  File extension(s): .pem
+  Macintosh file type code(s): n/a
+
+Person & email address to contact for further information: See Authors' Addresses section.
```

Comment:
IANA and the RFC Editor should see that section, right, not the consumer of the registry?

Reply:
AFAICT, this information doesn't go in the registry anyway, it just lives in the RFC.